### DISCUSSION TITLE
Hello, the latest LaTeX Workshop cannot preview PDF files and citations

### DISCUSSION THREAD

**Discussion by yexu55065** [James-Yu/LaTeX-Workshop#3368] [Q&A]
Hello, he latest LaTeX Workshop cannot preview PDF files and citations. Also I can't call the SnippetPanel via shortcut keys. Specific questions are as follows:

command 'latex-workshop.showSnippetPanel' not found
![Snipaste_2022-07-04_17-18-07](https://user-images.githubusercontent.com/20611525/177124376-4ea2bb39-d5bd-4397-82a7-dd5b6398581a.png)

![Snipaste_2022-07-04_17-21-34](https://user-images.githubusercontent.com/20611525/177124805-6f0f43c2-0c2a-4da8-8d16-f13cab9d9f58.png)

**Comment by James-Yu** [comment]
Please follow the issue template and provide necessary information.
